Multisig/Logic/Models/AppSettings.swift

```diff
@@ -151,7 +148,6 @@ struct PasscodeOptions: OptionSet {
     static let useBiometry = PasscodeOptions(rawValue: 1 << 0)
     static let useForLogin = PasscodeOptions(rawValue: 1 << 1)
     static let useForConfirmation = PasscodeOptions(rawValue: 1 << 2)
-    static let useForExportingKeys = PasscodeOptions(rawValue: 1 << 3)
```

Comment:
Can you add a comment that we had this flag as (1 << 3) value so that if we extend this parameter, existing app installation don't conflict with the deprecated value?